### ISSUE TITLE
Request for Documentation: AD Groups for Multi-Domains, Single forrests

### ISSUE THREAD

**Issue by matterdoc** [mattermost/docs#3025]
Mattermost user `katie.wiersgalla` from https://community-daily.mattermost.com has requested the following be documented:

```
@john.thompson I have completed the test for multi-domain, single forrest using a child trust and everything seems to work:
* Users from both domains can login
* Groups from both domains can be linked
* Group members from both domains get populated (**as long as the group is a Univeral group**, otherwise the `member` attributes are omitted by AD, for some reason)
```

See the original post [here](https://community-daily.mattermost.com/_redirect/pl/b5xrq38eht813dzw8dp6bwocar).

_This issue was generated from [Mattermost](https://mattermost.com) using the [Doc Up](https://github.com/jwilander/mattermost-plugin-docup) plugin._

**Comment by justinegeffen** [comment]
@wiersgallak, is [this section](https://docs.mattermost.com/deployment/sso-ldap.html#configure-ad-ldap-deployments-with-multiple-domains) sufficient to resolve this issue or does it need to be expanded?

**Comment by wiersgallak** [comment]
@justinegeffen Yeah I think this is a great start and we can collect more feedback from Customers if we need to expand more.

**Comment by cwarnermm** [comment]
Closing.
